src/notify.js

```diff
@@ -71,6 +83,11 @@ Notify.isSupported = function(perm) {
         throw new Error('You must only call this before calling Notification.requestPermission(), otherwise this feature detect would trigger an actual notification!');
     }
 
+    if (Notify.serviceWorkerSupported()) {
```

Comment:
To clarify, one would be used for detecting support for regular Web Notifications. The other would be to detect support for Notifications via Service Worker. They could be used entirely independently.